R/connect_user_totals_app.R

```diff
@@ -0,0 +1,320 @@
+# Connect-specific color mappings
```

Comment:
Per slack discussion, we may want to re-work this into a "Connect" report or a "Connect Users" report.